docs/recipes/php-cs-fixer/PrettierPHPFixer.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Fixer for using prettier-php to fix.
+ */
+final class PrettierPHPFixer implements FixerInterface {
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority() {
+        // should be absolute first
+        return 999;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens) {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isRisky() {
+        return false;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(SplFileInfo $file, Tokens $tokens) {
+        if (
+            0 < $tokens->count() &&
+            $this->isCandidate($tokens) &&
+            $this->supports($file)
+        ) {
+            $this->applyFix($file, $tokens);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName() {
+        return 'Prettier/php';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(SplFileInfo $file) {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    private function applyFix(SplFileInfo $file, Tokens $tokens) {
+        $tmpFile = $this->getTmpFile($file);
+        exec("yarn exec -- prettier --write --brace-style=1tbs $tmpFile");
```

Comment:
I'd stick with the default configuration (no options) in this example.

Reply:
thx for suggestion, yes this makes much more sense when this is going public 😄